### DISCUSSION TITLE
Sidebar within a page, instead of the site

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1252] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
According to the [documentation](https://fomantic-ui.com/modules/sidebar.html) sidebar works only for the full site, I mean within the body, instead of a given page.

```
<body>
  <div class="ui sidebar inverted vertical menu">
    <a class="item">
      1
    </a>
    <a class="item">
      2
    </a>
    <a class="item">
      3
    </a>
  </div>
  <div class="pusher">
    <!-- Site content !-->
  </div>
</body>
```

In my case I need a sidebar (with some search filters, not a items menu) only in a page of my web application.

Is it possible to reach that with the current sidebar module or have I to search for another solution?

**Comment by lubber-de** [comment, marked as answer]
A (FUI) sidebars purpose is to be displayed from offcanvas. A sidebar somwhere in the middle of the page cannot be properly shown the way the component is designed.
What about using [menu](https://fomantic-ui.com/collections/menu.html) instead ?

**Comment by smartm0use** [comment]
Ok, I will try to use menu for my purpose. Thank you for your prompt reply.
